Assets/Scenes/StartScreen.unity

```diff
@@ -2454,7 +2454,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
```

Comment:
I think rice league was the PvE mode wasn't it?  
The contract settings have only the first league as PvE - maybe this should be changed:
```
      - 0xEB6F3c59415efA4AA00e3D1be71827b52B0B4BB4
      - 0xC8A27e449f2e87B770f442A423B7591006847b3F
      - 0xC8A27e449f2e87B770f442A423B7591006847b3F
```

Reply:
> I think rice league was the PvE mode wasn't it? The contract settings have only the first league as PvE - maybe this should be changed:
> 
> ```
>       - 0xEB6F3c59415efA4AA00e3D1be71827b52B0B4BB4
>       - 0xC8A27e449f2e87B770f442A423B7591006847b3F
>       - 0xC8A27e449f2e87B770f442A423B7591006847b3F
> ```

I changed the value set when clicking on each league button on lines 7623 and 8413, those are the indexes for the array containing the matchmaker addresses, which i already removed one value leaving only 2 addresses 
https://github.com/pepemon-dao/pepemon-battle-degen/pull/77/files#diff-6292810005e4e08bbe7bbde94a65601b05d659d8dd5582cdc9162e049a14018aL7761

Reply:
so now the chad league uses index 1 and pepekarp uses index 0